xbmc/input/ButtonTranslator.cpp

```diff
@@ -1155,9 +1155,8 @@ int CButtonTranslator::GetFallbackWindow(int windowID)
     if (fallbackWindows[index].origin == windowID)
       return fallbackWindows[index].target;
   }
-  // for addon windows use WINDOW_ADDON_START
-  // because id is dynamic
-  if (windowID >= WINDOW_ADDON_START && windowID <= WINDOW_ADDON_END)
+  // for addon windows use WINDOW_ADDON_START because id is dynamic
+  if (windowID > WINDOW_ADDON_START && windowID <= WINDOW_ADDON_END)
```

Comment:
Right, indeed.
Now, with the change above, I wonder if we still need the changes below, as we'll recursively have:
WINDOW_ADDON_START+X -> WINDOW_ADDON_START -> -1, so no stack overflow, while we currently endlessly loop on WINDOW_ADDON_START...